### ISSUE TITLE
Show recent tabs

### ISSUE THREAD

**Issue by Bondrake** [mbnuqw/sidebery#475]
## Description

With a screen full of tabs open it can be hard to jump back to your recent ones sometimes. I would like it if there was an option for a marker that would show up on the __*x*__ most recent tabs. It could either be a glyph or a change to the whole tab row.

**Comment by Bondrake** [comment]
Would be especially useful if it were a sort of "ghost / fade", so when you click off a tab to a different one, the highlighting that's already on it just fades a bit. That could happen in a chain of a few tabs.

**Comment by Bondrake** [comment]
Another thought about this...

It would be really nice if there were a "recent tabs" panel. This would basically just be a view which you could enable which shows up as a panel that updates with the X most recent tabs showing up on it (either top to bottom or bottom to top). Clicking a tab there would jump you to the panel the tab is actually open in.

**Comment by Bondrake** [comment]
I use an MX Master mouse with the side scroll wheel to jump between panels. I have several panels open with, sometimes, hundreds of tabs. Having a recent tabs panel at the end would help immensely to navigate just between recent tabs across panels.

**Comment by the-a-man-006** [comment]
Yeah man, been searching for an extension that draws an MRU (Most Recently Used) ordered tabs list as a panel at the right side. Obviously, none exists. This should totally be a thing. It's not even that hard to implement, I'm guessing, compared to Sorting The Tree By MRU (https://github.com/mbnuqw/sidebery/issues/643#issuecomment-1536988134).

**Comment by the-a-man-006** [comment]
> Would be especially useful if it were a sort of "ghost / fade", so when you click off a tab to a different one, the highlighting that's already on it just fades a bit. That could happen in a chain of a few tabs.

TST Fade Old Tabs did just that, for TST, though, not Sidebery. I personally didn't like that concept. An independent sidebar panel for Recent Tabs is the best. Equally best would be whole (sub)tree-sort-by-recent. 

Meanwhile, I'm sticking with All Tabs Helper (this extension is made and maintained by a 66-year-old, and applying the CSS for dark mode was quite a nightmare too, personally). All Tabs Helper has this thing called a "Full View Window"; basically, an independent window which you can put Always On Top with a the PowerToys utility of the same name. My only issue with All Tabs Helper currently is that everytime you close the browser and reopen it, it forgets the Recent Tabs list entries... Makes it unusable if you're someone who keeps a hundred tabs opened...

[EDIT: History Bumper was a good alternative too; my only issue with it was, in the History window, you can only "open" or "open-in-new-tab" those links you'd recently visited, i.e., can't "switch to already opened tab (whose link/url is this)" through that window, so, basically, useless.].
